pkg/kbld/resources/file_sources.go

```diff
@@ -63,7 +63,7 @@ func (s HTTPFileSource) Bytes() ([]byte, error) {
 
 	defer resp.Body.Close()
 
-	result, err := ioutil.ReadAll(resp.Body)
+	result, err := io.ReadAll(resp.Body)
```

Comment:
Here also should we use the `os` package only as it is already imported so we can ignore importing `io`?